flood_ml/constants.py

```diff
@@ -0,0 +1,10 @@
+"""Constant definitions for Flood CNN."""
```

Comment:
Some notes on package structure. I'd suggest a directory structure like
```
flood_ml/
  - requirements.txt
  - setup.py
  - flood_ml/
    - __init__.py
    - constants.py
    - data_utils.py
  - tests/
    - test_data_utils.py
```
That will be compatible with the conventions tools like pytest use for finding tests and create a clearly defined `flood_ml` package with attendant requirements.txt and/or setup.py files.

Reply:
Resolving this, since there's another thread about the package structure.